### PR TITLE
Afs942 update manifest manager

### PR DESCRIPTION
**Pull request by njkim**
### Types of changes

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

Update images service manager archesproject/arches-for-science#942
1. Add placeholder for attribution logo and update the logic to get logo from url
2. Update the metadata when it is not added but filled and allow to delete metadata when deleted
3. Use the first image name as a service title by default

### Issues Solved

#8973

### Checklist

-   [ ] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @

### CODE REVIEW

File: arches/app/views/manifest_manager.py [hunk 1]
```diff
@@ -195,14 +195,21 @@ def change_canvas_label(manifest, canvas_id, label):
...
         attribution = request.POST.get("manifest_attribution", "")
         logo = request.POST.get("manifest_logo", "")
-        desc = request.POST.get("manifest_description")
+        desc = request.POST.get("manifest_description", "")
         operation = request.POST.get("operation")
         manifest_url = request.POST.get("manifest")
         canvas_label = request.POST.get("canvas_label")
         canvas_id = request.POST.get("canvas_id")
         metadata = request.POST.get("metadata")
+        print(metadata)
```
Comment (chiatt) [inline arches/app/views/manifest_manager.py:212]: Remove

**Review by chiatt — CHANGES_REQUESTED** [review]
This looks really good except there is a print statement to remove and I noticed that the dirty state does not change if the only thing that is modified is the attribution url.

File: arches/app/views/manifest_manager.py (hunk 1 above)
Comment (njkim) [reply arches/app/views/manifest_manager.py]: Thanks.
The print statement has been removed.
Also, dirty state is restored by updating the part to display logo.

**Review by chiatt — APPROVED** [review]
Looks good and addresses all issues in ticket.

**Pull request merged**